github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -205,38 +205,6 @@ presubmits:
       timeout: 7h
       grace_period: 5m
     max_concurrency: 10
-    labels:
```

Comment:
Why did we have all those twice?

Reply:
It's just git interpretation of the change, since I am removing multus and it's very similar it thinks I have change it.

Reply:
Oh right :man_facepalming: